src/main/webapp/content/css/overrides.css

```diff
@@ -0,0 +1,12 @@
+/* navbar */
+
+.jh-navbar {
+  background-color: #222035 !important;
```

Comment:
Can we not use this !important annotation when making changes, but rather directly change [.jh-navbar](https://github.com/python-sinhala-education-society/OpenLearnr/blob/b3da28ad4b644f29cec863dc96d79cfa779742b0/src/main/webapp/app/layouts/navbar/navbar.scss#L8) and [.hipster](https://github.com/python-sinhala-education-society/OpenLearnr/blob/b3da28ad4b644f29cec863dc96d79cfa779742b0/src/main/webapp/app/home/home.scss#L5). It makes it difficult to maintain the code in the future; https://css-tricks.com/when-using-important-is-the-right-choice/

Reply:
changed css to navbar.scss and deleted overrides.css file ;) change come in the next sync